ClearBank.ApiTest/ClearBank.ApiTest/Features/DeckOfCards.feature

```diff
@@ -0,0 +1,15 @@
+﻿Feature: DeckOfCards
+
+Scenario Outline: Remaining cards correctly updated after drawing
+	Given I create a new deck of cards
+	When I draw 5 cards
```

Comment:
The test actually asked to draw from the deck 1 time for each scenario but have different draw values per scenario. 

So this first `When I draw 5 cards` line is not needed.